tizen-api/include/nnstreamer.h

```diff
@@ -215,8 +216,8 @@ typedef void (*nns_sink_cb)
 int nns_pipeline_construct (const char *pipeline_description, nns_pipeline_h *pipe);
 
 /**
- * @brief Destroy the pipeline
- * @detail Use this function to destroy the pipeline constructed with nns_construct_pipeline.
+ * @brief Destroys the pipeline
+ * @detail Uses this function to destroy the pipeline constructed with nns_construct_pipeline.
```

Comment:
I do not know what is the convention for Tizen APIs used here, but other comments mentioned function names seems to be followed by (). Is it ok without () here?

Reply:
According to the guide, a verb after `@brief` should be a third person singular format without `()`. You can see a code of autofill module, which is recently merged into Tizen.
* https://git.tizen.org/cgit/platform/core/uifw/autofill/tree/include/autofill.h?h=tizen#n208